


for i in range(0,10):
    try:
        print(i)
        if(i==4):
            print(ijk)
    except Exception as err:
        print(err)

#让异常的程序继续执行下去

